src/circular-progress/CircularProgress.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+  useProgressState,
+  ProgressInitialState,
+  ProgressStateReturn,
+} from "@renderlesskit/react";
+import * as React from "react";
+
+import { BoxProps } from "../box";
+import { createContext, runIfFn } from "../utils";
+import { forwardRefWithAs } from "../utils/types";
+import { CircularProgressBar } from "./CircularProgressBar";
+import { CircularProgressWrapper } from "./CircularProgressWrapper";
+
+export type CircularProgressContext = Pick<CircularProgressProps, "size"> & {
+  state: ProgressStateReturn;
+};
+
+const [
+  CircularProgressProvider,
+  useCircularProgressContext,
+] = createContext<CircularProgressContext>({
+  name: "CircularProgressContext",
+  strict: false,
+});
+
+export { useCircularProgressContext };
+
+type CircularProgressRenderProps = {
+  children?:
+    | (({ ...state }: ProgressStateReturn) => JSX.Element)
```

Comment:
`((state: ProgressStateReturn) => JSX.Element)`

Reply:
Why not spreading them ?
Whats the issue ?

Reply:
Got it !